wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -145,13 +149,21 @@ public UsersWorkspaceImpl createWorkspace(WorkspaceConfig config, String owner,
     }
 
     /**
-     * Gets workspace by its id.
+     * Gets workspace by composite key.
+     *
+     * <p> Key rules:
+     * <ul>
+     * <li>If it doesn't contain <b>:</b> character then that key is id(e.g. workspace123456)
+     * <li>If it contains <b>:</b> character then that key is combination of user name and workspace name
+     * <li><b></>:workspace_name</b> is valid abstract key (e.g. che won't have user concept)
```

Comment:
> che won't have user concept

Looks like it is false for now. Maybe it would be better to describe that in case of `:workspace_name` user will be taked from EnviromentContext?